docs/yamlpages/cookbook.rst

```diff
@@ -7,6 +7,169 @@ Having all the options laid out in front of you is good for advanced users, but
 Below is the YANK YAML Cookbook.
 A series of examples that may help you understanding how to put together these options found on the other pages.
 
+----
+
+.. _yaml_ex_implicit:
+
+Example: Implicit Solvent Binding Simulation:
+=============================================
+
+This example sets up para-xylene binding to T4-Lysozyme in implciit solvent. 
+We reccommend these settings as a good baseline simulation setup.
+Although you do not have to set all these options (e.g. the ligand may be a pdb file, not just a SMILES string), many 
+of these options are good stock settings to remember.
+
+In this example:
+
+* Targeting folders for input files
+* Configuring Output files
+* Setting good stock options for implicit simulations
+* Setting up simple protein/ligand binding simulation
+* NVT ensemble
```

Comment:
Default pressure value is `1 * atmosphere`, not `null` so I think this is still NPT.